tests/reporting/test_reporting.py

```diff
@@ -1,558 +1,241 @@
-    assert completeSets.publicSellCompleteSets(newMarket.address, 10, sender=tester.k1)
-    assert cash.balanceOf(newMarket.getReportingWindow()) > 0
-
-    # We'll do a designated report in the new market based on the makeReport param used for the forking market
-    proceedToDesignatedReporting(localFixture, universe, newMarket, [0,numTicks])
-    if (makeReport):
-        localFixture.designatedReport(newMarket, [0,numTicks], tester.k0)
-
-    # We proceed the standard market to the FORKING state
-    proceedToForking(localFixture, universe, market, makeReport, 1, 2, 3, [0,numTicks], [numTicks,0], 2, [numTicks,0], [0,numTicks], [numTicks,0])
-
-    # The market we created is now awaiting migration
-    assert newMarket.getReportingState() == localFixture.contracts['Constants'].AWAITING_FORK_MIGRATION()
-
-    # If we attempt to migrate now it will not work since the forking market is not finalized
-    with raises(TransactionFailed, message="Migration cannot occur until the forking market is finalized"):
-        newMarket.migrateThroughOneFork()
-
-    # We'll finalize the forking market
-    finalizeForkingMarket(localFixture, universe, market, finalizeByMigration, tester.a1, tester.k1, tester.a0, tester.k0, tester.a2, tester.k2, [0,numTicks], [numTicks,0])
+def test_designatedReportHappyPath(localFixture, universe, market):
+    # proceed to the designated reporting period
+    proceedToDesignatedReporting(localFixture, market)
 
-    # Now we can migrate the market to the winning universe. We also confirm that migration will move fees
-    windowCashBalance = cash.balanceOf(newMarket.getReportingWindow())
-    with TokenDelta(cash, -windowCashBalance, newMarket.getReportingWindow(), "Fork migration did not trigger window fee migratgion"):
-        assert newMarket.migrateThroughOneFork()
+    # an address that is not the designated reporter cannot report
+    with raises(TransactionFailed):
+        market.doInitialReport([0, market.getNumTicks()], False, sender=tester.k1)
 
-    assert cash.balanceOf(newMarket.getReportingWindow()) == windowCashBalance
+    # Reporting with an invalid number of outcomes should fail
+    with raises(TransactionFailed):
+        market.doInitialReport([0, 0, market.getNumTicks()], False)
+
+    # do an initial report as the designated reporter
+    initialReportLog = {
+        "universe": universe.address,
+        "reporter": bytesToHexString(tester.a0),
+        "market": market.address,
+        "amountStaked": localFixture.contracts["Constants"].DEFAULT_DESIGNATED_REPORT_STAKE(),
+        "isDesignatedReporter": True,
+        "payoutNumerators": [0, market.getNumTicks()]
+    }
+    with AssertLog(localFixture, "InitialReportSubmitted", initialReportLog):
+        assert market.doInitialReport([0, market.getNumTicks()], False)
+
+    # the market is now assigned a fee window
+    assert market.getFeeWindow()
+    feeWindow = localFixture.applySignature('FeeWindow', market.getFeeWindow())
```

Comment:
Very minor, but you do `market.getFeeWindow` twice in a row here (and same in other tests).  Since we know that executing EVM code is slow, may be worth a local variable.